Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java

```diff
@@ -159,4 +168,32 @@ private static void extractTarFileToFolder(File file, File destinationFolder) {
         tarGZipUnArchiver.setSourceFile(file);
         tarGZipUnArchiver.extract("", destinationFolder);
     }
+
+    private static String fetchLatestStableChromeDriverVersion() {
+
+        String version = null;
+        try {
+
+            URL url = new URL(CHROME_DRIVER_LATEST_RELEASE_URL);
+            URLConnection urlConnection = url.openConnection();
+
+            BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(urlConnection.getInputStream()));
```

Comment:
Please use a `try-with-resources` statement instead of open and closing the streams by yourself.

Reply:
I can add this but I don't think it really matters in lifecycle of the test run.

This part is executed once at the beginning of the selenium tests and then it's done. If this fails the test run is over anyway.